packages/charts/src/components/tooltip/components/_tooltip.scss

```
@@ -223,10 +148,7 @@ $tableRowHoverSelectedColor: tintOrShade($euiFocusBackgroundColor, 0, 10%);
     position: relative;
 
     &--highlighted {
-      .echTooltip__tableCell:not(.echTooltip__colorCell),
-      .echTooltip__colorCell .echTooltip__colorStrip--spacer {
-        background-color: transparentize($euiColorLightShade, 0.5);
-      }
+      background-color: transparentize($euiColorLightShade, 0.5);
```

This does not work because the `__tableRow` has `display: contents` for sizing the grid columns. So it must be applied to the children.

```suggestion
      .echTooltip__tableCell:not(.echTooltip__colorCell),
      .echTooltip__colorCell .echTooltip__colorStrip--spacer {	
        background-color: transparentize($euiColorLightShade, 0.5);	
      }
```